webapp/backend/core/ml_layer.py

```diff
@@ -53,7 +51,7 @@ def transcribe_audio_chunk(audio_file: str, base_url: str) -> str:
         bytes_data = base64.b64encode(audio_bytes).decode()
 
         #TODO: request the actual round here
-        payload = {"data": bytes_data, "filename": os.path.basename(audio_file), "current_round": current_round}
+        payload = {"data": bytes_data, "filename": os.path.basename(audio_file), "current_round": 4}
```

Comment:
I believe this function isn't used in the current flow so is a a bit inconsequential but it should mimic the other ones as well so let's pass it into the function also. More importantly if we aren't using this function anywhere in the actual web app backend let's just remove it so it's code we don't need